### PR TITLE
Sentence case for section titles

### PR DESCRIPTION
**Pull request by RichardHoch**
Correcting the titles of sections to use Sentence Case.

### CODE REVIEW

File: docs/topics_5/cli-args.adoc [hunk 1]
```diff
@@ -60,13 +60,22 @@ Do not overwrite a report output directory that contains important information.
...
+|--sourceMode |Flag to indicate that the application to be evaluated contains source files rather than compiled binaries. See the xref:input_file_type_arguments[Input file argument tables] for details.
+|--target |A space-delimited list of one or more target technologies, servers, platforms, or frameworks to migrate to. This argument, in conjunction with the `--source` argument, helps to determine which rulesets are used. Use the `--listTargetTechnologies` argument to list all available targets. See xref:target_argument[Set the target technology] for more information.
+|--userIgnorePath |Specify a location, in addition to `${user.home}/.mta/ignore/`, for {ProductShortName} to identify files that should be ignored. For additional information on formatting these files, see xref:exclude_files_and_packages[Configure {ProductShortName} to exclude files and packages].
+>>>>>>> Sentence case for section titles
```
Comment (apinnick) [inline docs/topics_5/cli-args.adoc:71]: You've got a merge conflict marker here.

File: docs/topics_5/create-basic-xml-rule.adoc [hunk 2]
```diff
@@ -93,7 +93,9 @@ Individual rules are contained within the `<rules>` element. They comprise one o
 See the link:http://windup.jboss.org/schema/windup-jboss-ruleset.xsd[XML rule schema] for valid rule syntax.
 
 [id='create_when_condition_{context}']
+<<<<<<< HEAD
 === Creating a `<when>` Condition
+>>>>>>> Resolve conflicts
```
Comment (apinnick) [inline docs/topics_5/create-basic-xml-rule.adoc:98]: merge conflict marker

File: docs/topics_5/create-basic-xml-rule.adoc [hunk 3]
```diff
@@ -140,7 +142,9 @@ The XML rule `<when>` element tests for a condition. The following is a list of
 The specific syntax is dependent on whether you are creating a rule to evaluate Java class, an XML file, a project, or file content.
 
 [id='create_perform_action_{context}']
+<<<<<<< HEAD
 === Creating a `<perform>` Action
+>>>>>>> Resolve conflicts
```
Comment (apinnick) [inline docs/topics_5/create-basic-xml-rule.adoc:148]: merge conflict marker

File: docs/topics_5/web-openshift-deploy.adoc [hunk 4]
```diff
@@ -1,16 +1,23 @@
...
+= Deploying the {ProductShortName} application
+
+. Download the {WebName} from the link:https://developers.redhat.com/products/mta/download[{ProductShortName} download page].
 
-. Download the {WebName} from the link:https://developers.redhat.com/products/mta/download[{ProductShortName} Download page].
 . Extract the ZIP file to a directory of your choice.
 +
+<<<<<<< HEAD
 The path to the directory created by unzipping this file is referred to as `<MTA_HOME>` throughout this guide.
+=======
```
Comment (apinnick) [inline docs/topics_5/web-openshift-deploy.adoc:12]: lots of merge conflict markers

File: docs/topics/known-issues.adoc [hunk 5]
```diff
@@ -1,4 +1,12 @@
+<<<<<<< HEAD:docs/topics/known-issues.adoc
 [[known_issues]]
 = Known {ProductShortName} Issues
+=======
+// Module included in the following assemblies:
+// * docs/cli-guide_5/master.adoc
+// * docs/maven-guide_5/master.adoc
+[id='known_issues_{context}']
+= Known {ProductShortName} issues
+>>>>>>> fix:docs/topics_5/known-issues.adoc
```
Comment (apinnick) [inline docs/topics/known-issues.adoc:10]: Merge conflict marker

File: docs/topics_5/cloud-story-points-release.adoc [hunk 6]
```diff
@@ -1,4 +1,4 @@
 [id='cloud_story_points_{context}']
-= Technology Summary Report Now Includes Potential and Cloud-Mandatory Tasks
```
Comment (apinnick) [inline docs/topics_5/cloud-story-points-release.adoc:2]: You don't have to work on anything that goes into release notes. The fact that this file does not have an assembly comment indicates that it's not in one of our current assemblies. Also, it sounds like a release note.

File: docs/topics_5/create-first-xml-rule.adoc [hunk 7]
```diff
@@ -190,7 +190,7 @@ The rule is now complete and should look like the following example.
 ----
 
 [discrete]
-== Install the Rule
+== Install the rule
```
Comment (apinnick) [inline docs/topics_5/create-first-xml-rule.adoc:193]: I mjust have missed these. They shoujld be gerunds

File: docs/topics_5/rules-categories.adoc [hunk 8]
```diff
@@ -26,7 +31,11 @@ You can add a custom category to the rule category file.
 * `name`: The display name of the category.
 * `description`: The description of the category.
 +
+<<<<<<< HEAD
 .Custom rule category example
+=======
+.Example Custom rule category
+>>>>>>> fix
```
Comment (apinnick) [inline docs/topics_5/rules-categories.adoc:38]: merge conflict markers to remove

File: docs/topics_5/rules-categories.adoc [hunk 9]
```diff
@@ -42,15 +51,25 @@ You can add a custom category to the rule category file.
 This category is ready to be referenced by {ProductShortName} rules.
 
 [id='assign_custom_category_{context}']
+<<<<<<< HEAD
 == Assigning a Rule to a Custom Category
+=======
+[discrete]
+== Assigning a rule to a custom category
```
Comment (apinnick) [inline docs/topics_5/rules-categories.adoc:58]: more markers

File: docs/topics_5/validation-report-errors.adoc [hunk 10]
```diff
@@ -1,7 +1,11 @@
 // Module included in the following assemblies:
 // * docs/rules-development-guide_5/master.adoc
 [id='validation_report_errors_{context}']
+<<<<<<< HEAD
 = Validation report error messages
+=======
+= Validation report errors
+>>>>>>> fix
```
Comment (apinnick) [inline docs/topics_5/validation-report-errors.adoc:8]: merge markers

File: docs/topics_5/validation-report-understanding.adoc [hunk 11]
```diff
@@ -1,23 +1,40 @@
 // Module included in the following assemblies:
 // * docs/rules-development-guide_5/master.adoc
 [id='validation_report_understanding_{context}']
+<<<<<<< HEAD
 = Creating a validation report
+=======
+= Understanding the validation report
+>>>>>>> fix
```
Comment (apinnick) [inline docs/topics_5/validation-report-understanding.adoc:8]: merge conflict marker

File: docs/topics/known-issues.adoc [hunk 12]
```diff
@@ -1,4 +1,7 @@
-[[known_issues]]
-= Known {ProductShortName} Issues
+// Module included in the following assemblies:
```
Comment (apinnick) [inline docs/topics/known-issues.adoc:1]: You changed the wrong file. We're not using the 'topics' folder. Just topics_5

File: docs/topics_5/exclude-files-and-packages.adoc [hunk 13]
```diff
@@ -1,10 +1,10 @@
 // Module included in the following assemblies:
 // * docs/cli-guide_5/master.adoc
 [id='exclude_files_and_packages_{context}']
-= Configuring {ProductShortName} to Exclude Files and Packages
+= Configuring {ProductShortName} to exclude files and packages
```
Comment (apinnick) [inline docs/topics_5/exclude-files-and-packages.adoc:5]: Just a comment about word order. If "Excluding packages" precedes "Excluding files", might be better to reverse the order in this title.

File: docs/topics_5/plugin-components.adoc [hunk 14]
```diff
@@ -1,23 +1,23 @@
 // Module included in the following assemblies:
 // * docs/plugin-guide_5/master.adoc
 [id='plugin_components_{context}']
-= About {PluginNameTitle} Components
+= About {PluginNameTitle} components
 
 The following components are available in the {ProductShortName} perspective when using the {PluginName} to analyze projects.
 
-Issue Explorer:: This view allows you to explore the {ProductShortName} issues for projects that have been analyzed.
+Issue explorer:: This view allows you to explore the {ProductShortName} issues for projects that have been analyzed.
```
Comment (apinnick) [inline docs/topics_5/plugin-components.adoc:8]: I checked the UI. Issue Explorer and the rest of these components are still "title case", so they have to be reverted. The plugin UI has not been updated for a while. Same for Issue Details and MTA Report. They have to be reverted.

BTW, whenever you change UI elements, you should **check the screenshots** before making changes to the text.

File: docs/topics_5/plugin-create-config.adoc [hunk 15]
```diff
@@ -1,14 +1,14 @@
 // Module included in the following assemblies:
 // * docs/plugin-guide_5/master.adoc
 [id='create_run_config_{context}']
-= Creating an {ProductShortName} Run Configuration
+= Creating an {ProductShortName} run configuration
 
 {ProductShortName} run configurations can be created using the {ProductShortName} button (image:windup.png[{ProductShortName} button]). A run configuration specifies the project to analyze, migration path, and additional options for the execution. You can create multiple run configurations, and each must have a unique name.
```
Comment (apinnick) [inline docs/topics_5/plugin-create-config.adoc:6]: Remove space at beginning of line 6

File: docs/topics_5/plugin-create-config.adoc [hunk 16]
```diff
@@ -1,14 +1,14 @@
...
 
 [discrete]
 == Input
 
-Migration Path::
+Migration path::
 Select a migration path, which determines which {ProductShortName} rulesets are used. The migration path defaults to *Anything to EAP 7*, but can be changed to any supported migration path.
```
Comment (apinnick) [inline docs/topics_5/plugin-create-config.adoc:12]: This has to be checked against screenshots

File: docs/topics_5/plugin-resolve-issues.adoc [hunk 17]
```diff
@@ -1,24 +1,24 @@
...
 [id='resolve_issues_{context}']
-= Resolving {ProductShortName} Issues
+= Resolving {ProductShortName} issues
 
 You can resolve {ProductShortName} issues by updating the code manually or by applying a quick fix when available.
 
-== Resolving an Issue Manually
+== Resolving an issue manually
 
 Review the {ProductShortName} issue details and additional resources and update the source code as necessary. When you update a line of code marked as an {ProductShortName} issue, the {ProductShortName} issue will be marked with the stale icon (image:stale_issue.gif[Stale issue icon]) until the next time that {ProductShortName} is run on the project.
```
Comment (apinnick) [inline docs/topics_5/plugin-resolve-issues.adoc:10]: Remove extra space at beginning of lines 10 and 12.

File: docs/topics_5/web-add-applications.adoc [hunk 18]
```diff
@@ -1,20 +1,20 @@
...
 // * docs/web-console-guide_5/master.adoc
 [id='add_applications_{context}']
-= Adding Applications to a Project
+= Adding applications to a project
 
 Specify the applications to add to this project. You can upload applications or register a server path that contains applications.
 
-.Add Applications
-image::web-add-apps.png[Add Applications]
+.Add applications
```
Comment (apinnick) [inline docs/topics_5/web-add-applications.adoc:8]: Add < Addubg

File: docs/topics_5/web-add-project.adoc [hunk 19]
```diff
@@ -1,16 +1,16 @@
...
 
 .Home
-image::web-no-projects.png[Home: No Projects]
+image::web-no-projects.png[Home: no projects]
 
 Click the *New Project* button to launch the wizard to set up a new migration project.
 
-.Add Project
-image::web-add-project.png[Add Project]
+.Add project
```
Comment (apinnick) [inline docs/topics_5/web-add-project.adoc:13]: Add > Adding a

File: docs/topics_5/web-custom-rules.adoc [hunk 20]
```diff
@@ -52,5 +52,5 @@ Click *Add* to register the labels.
 
 Clicking on the _Labels_ icon displays the contents of the `labels` file.
 
-.Viewing the `Labels` File Contents
-image::web-labels-file-view.png[View Labels File Contents]
+.Viewing the `labels` file contents
+image::web-labels-file-view.png[View labels file contents]
```
Comment (apinnick) [inline docs/topics_5/web-custom-rules.adoc:56]: View > Viewing the `labels` file

File: docs/topics_5/web-execute.adoc [hunk 21]
```diff
@@ -1,7 +1,7 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='execute_{context}']
-= Running an Analysis
+= Running an analysis
 
 From the analysis configuration page, click the *Save & Run* button to execute the analysis.
```
Comment (apinnick) [inline docs/topics_5/web-execute.adoc:6]: Line 8: to execute > to run

File: docs/topics_5/web-execution-config.adoc [hunk 22]
```diff
@@ -1,14 +1,14 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='analysis_configuration_{context}']
-= Configuring the Analysis
+= Configuring the analysis
```
Comment (apinnick) [inline docs/topics_5/web-execution-config.adoc:4]: Change to: Configuring an analysis

File: docs/topics_5/web-execution-config.adoc [hunk 23]
```diff
@@ -1,14 +1,14 @@
...
 
 Configure the settings for the analysis, such as the transformation path, packages, and other advanced options.
 
-.Configuring Analysis
-image::web-configure-analysis.png[Configure Analysis]
+.Configuring analysis
+image::web-configure-analysis.png[Configure analysis]
```
Comment (apinnick) [inline docs/topics_5/web-execution-config.adoc:9]: Configuring an analysis. Change both image caption and image name

File: docs/topics_5/web-execution-config.adoc [hunk 24]
```diff
@@ -1,14 +1,14 @@
...
 
-Transformation Path::
+Transformation path::
```
Comment (apinnick) [inline docs/topics_5/web-execution-config.adoc:12]: Check these labels against the screenshots to make sure that they are sentence case. The web console is very inconsistent. We must always record the current state of the UI.

File: docs/topics_5/web-execution-config.adoc [hunk 25]
```diff
@@ -44,10 +44,10 @@ Project custom rules are maintained using the controls below.
 The *Add* button allows the user to select the rules files ([x-]`.windup.xml` or [x-]`.mta.xml`).
 Selected (checked) rules are shown in the right-hand panel and will be executed during the analysis.
 
-.Custom Rules
-image::web-custom-rules-project.png[The Custom Rules dialog]
+.Custom rules
+image::web-custom-rules-project.png[The custom rules dialog]
```
Comment (apinnick) [inline docs/topics_5/web-execution-config.adoc:49]: Change image name to "Custom rules"" so that it matches its caption.

File: docs/topics_5/xml-rule-perform-syntax.adoc [hunk 26]
```diff
@@ -77,12 +77,12 @@ of="MySeamRule"
 ----
 |====
 
-=== <classification> Child Elements
+=== <classification> Child elements
```
Comment (apinnick) [inline docs/topics_5/xml-rule-perform-syntax.adoc:80]: Child > child

File: docs/topics_5/xml-rule-perform-syntax.adoc [hunk 27]
```diff
@@ -315,7 +315,7 @@ The following is an example of rule that defines an XSLT action.
 </rule>
 ----
 
-=== <xslt> Element Attributes
+=== <xslt> Element attributes
```
Comment (apinnick) [inline docs/topics_5/xml-rule-perform-syntax.adoc:318]: Element > element

**Comment by apinnick** [comment]
@RichardHoch I've reviewed again. Please see my comments.

File: docs/topics_5/exclude-files-and-packages.adoc (hunk 13 above)
Comment (RichardHoch) [reply docs/topics_5/exclude-files-and-packages.adoc:5]: Done.

File: docs/topics_5/plugin-components.adoc (hunk 14 above)
Comment (RichardHoch) [reply docs/topics_5/plugin-components.adoc:8]: Done.

File: docs/topics_5/plugin-create-config.adoc (hunk 15 above)
Comment (RichardHoch) [reply docs/topics_5/plugin-create-config.adoc:6]: Just looks like space.

File: docs/topics_5/plugin-create-config.adoc (hunk 16 above)
Comment (RichardHoch) [reply docs/topics_5/plugin-create-config.adoc:12]: There was no available screenshot. In context, it seems likely to be a UI label, so I reverted it.

File: docs/topics_5/plugin-resolve-issues.adoc (hunk 17 above)
Comment (RichardHoch) [reply docs/topics_5/plugin-resolve-issues.adoc:10]: They just look spaces.

File: docs/topics_5/web-add-applications.adoc (hunk 18 above)
Comment (RichardHoch) [reply docs/topics_5/web-add-applications.adoc:8]: Done.

File: docs/topics_5/web-add-project.adoc (hunk 19 above)
Comment (RichardHoch) [reply docs/topics_5/web-add-project.adoc:13]: Done.

File: docs/topics_5/web-execute.adoc (hunk 21 above)
Comment (RichardHoch) [reply docs/topics_5/web-execute.adoc:6]: Done.

File: docs/topics_5/web-execution-config.adoc (hunk 23 above)
Comment (RichardHoch) [reply docs/topics_5/web-execution-config.adoc:9]: Done.

File: docs/topics_5/web-execution-config.adoc (hunk 24 above)
Comment (RichardHoch) [reply docs/topics_5/web-execution-config.adoc:12]: Done.

File: docs/topics_5/web-execution-config.adoc (hunk 25 above)
Comment (RichardHoch) [reply docs/topics_5/web-execution-config.adoc:49]: Done. And checked against screenshot.

Comment (apinnick) [reply docs/topics_5/web-execution-config.adoc:49]: But why is the image name "The custom rules dialog"? THat does not sound good.

File: docs/topics_5/xml-rule-perform-syntax.adoc (hunk 26 above)
Comment (RichardHoch) [reply docs/topics_5/xml-rule-perform-syntax.adoc:80]: Done.

File: docs/topics_5/review-reports.adoc [hunk 28]
```diff
@@ -276,12 +276,12 @@ The dependency graph may be adjusted by using any of the following.
 * Clicking on any item in the legend will enable or disable all items of the selected type. For instance, selecting the embedded WARs icon will disable all embedded WARs if these are enabled, and will enable these dependencies if they are disabled.
 
 [id='shared_archives_{context}']
-== Archives Shared by Multiple Applications
+== Shared Libraries: Archives shared by multiple applications
```
Comment (RichardHoch) [inline docs/topics_5/review-reports.adoc:279]: Hard item to caption properly. 
               
image

Comment (apinnick) [reply docs/topics_5/review-reports.adoc:279]: If you are documenting a concept, you use sentence case. If you are documenting the UI itself, you document the UI.

This is a concept. You do not have to slavishly follow what the UI says. In fact, in many cases, the UI might be too terse or ambiguous to understand easily, so you would have to explain the UI clearly (and then submit UI bug to fix the English). So use sentence case. Doesn't matter what the UI says. You're documenting a concept here.

**Comment by RichardHoch** [comment]
Changes from yesterday: 1. Resolved the issue about the correct title for the section containing these reports: 
image
Solution: Used original heading in sentence case,
2. Found a few headings with verbs that needed to be changed to gerunds. 
I think this PR is ready to be merged.

File: docs/topics_5/cli-args.adoc [hunk 29]
```diff
@@ -37,7 +37,7 @@ To run the {ProductShortName} command without prompting, for example when execut
 |--includeTags| A space-delimited list of tags to use. When specified, only rules with these tags will be processed. To see the full list of tags, use the `--listTags` argument.
 |--input |A space-delimited list of the path to the file or directory containing one or more applications to be analyzed. This argument is required.
 |--install |Specify add-ons to install. The syntax is `<GROUP_ID>:<ARTIFACT_ID>[:<VERSION>]`. For example, `--install core-addon-x` or `--install org.example.addon:example:1.0.0`.
-|--keepWorkDirs| Flag to instruct {ProductShortName} to not delete temporary working files, such as the graph database and extracted archive files. This is useful for debugging purposes.
+|--keepWorkDirs| Flag to instruct {ProductShortName} to not delete temporary working files, such as the graph database and unzipped archives. This is useful for debugging purposes.
```
Comment (apinnick) [inline docs/topics_5/cli-args.adoc:40]: Revert this change. I made it this morning

Comment (RichardHoch) [reply docs/topics_5/cli-args.adoc]: Fixed.
